header.php

```diff
@@ -122,6 +125,10 @@ function wp_body_open() {
 </div>
 </header>
 <?php
+
+include "inc/information-banner.php";
```

Comment:
Please could we move this file to partials directory with other page elements

Reply:
Done